core/runtime/jsruntime.rs

```diff
@@ -454,6 +456,10 @@ pub struct RuntimeOptions {
   /// An optional instance of `FeatureChecker`. If one is not provided, the
   /// default instance will be created that has no features enabled.
   pub feature_checker: Option<Arc<FeatureChecker>>,
+
+  /// A callback that can be used to validate import attributes received at
+  /// the import site.
```

Comment:
Maybe this should say something about how providing `None` will use the default validator and describe how the default validator works? Otherwise it kind of seems like providing `None` will cause import attributes to not be validated.

Reply:
Sure, I'll update that